.babelrc

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
This could be replaced with WordPress' presets:

https://github.com/WordPress/gutenberg/blob/master/package.json#L99-L102

Reply:
Nice!